Changelog.md

```diff
@@ -17,6 +17,7 @@ Language Features:
 
 Compiler Features:
  * General: Generate JSON representations of Yul ASTs only on demand to reduce memory usage.
+ * SMTChecker: Z3 is now a runtime dependency, not a build dependency (except for emscripten build).
```

Comment:
Changelog entry needs to move to 0.8.29 - but then I hope we can merge this soon!

Reply:
Changelog entry has been moved.